### PR TITLE
feat(#255): add namespace for workers in k8s

### LINKED ISSUE #255: Config the namespace for workers in k8s cloud
**What happened**
Currently, all workers are created in the same namespace with Cyclone server. Need to make this configurable.

**What you expected to happen**
The project can specify the namespace for workers used by all pipelines in this project.

### PR DESCRIPTION
**Pull request by supereagle**
## Fixes #255

**Changes**

  - [x] add namespace for workers in k8s

### CODE REVIEW

File: cloud/k8s.go [hunk 1]
```diff
@@ -180,7 +180,17 @@ func (cloud *K8SCloud) CanProvision(quota Quota) (bool, error) {
 
 // Provision returns a worker if the cloud can provison
 func (cloud *K8SCloud) Provision(id string, wopts WorkerOptions) (Worker, error) {
-	can, err := cloud.CanProvision(wopts.Quota)
+	var cp *K8SCloud
```
Comment (bbbmj) [inline cloud/k8s.go]: What does cp mean?

Comment (supereagle) [reply cloud/k8s.go]: `cp` is short for `cloud provider`. It is just a local variable, so the the abbreviation.

**Pull request merged**